src/test/java/org/gridsuite/directory/server/utils/MatcherJson.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
We might use this class out of the tests ?
Maybe move it into test package ?

Reply:
Already in the `test` package ?

Reply:
Ah yes sorry. So fine for me